src/Windowing/Silk.NET.Windowing.Glfw/GlfwWindow.cs

```diff
@@ -239,17 +240,14 @@ public override void SetWindowIcon(ReadOnlySpan<RawImage> icons)
                 {
                     var icon = icons[i];
                     // ReSharper disable once StackAllocInsideLoop
-                    var iconMemory = stackalloc byte[icon.Pixels.Length];
+                    Span<byte> iconMemory = stackalloc byte[icon.Pixels.Length];
                     images[i] = new Image
                     {
                         Width = icon.Width, Height = icon.Height,
-                        Pixels = iconMemory
+                        Pixels = (byte*)Unsafe.AsPointer(ref iconMemory[0])
                     };
-
-                    for (var j = 0; j < icon.Pixels.Length; j++)
-                    {
-                        iconMemory[j] = icon.Pixels[j];
-                    }
+                    
+                    icon.Pixels.Span.CopyTo(iconMemory);
                 }
 
                 _glfw.SetWindowIcon(_glfwWindow, icons.Length, images);
```

Comment:
I think at this point the `stackalloc`ed memory will be released, so the `images[i].Pixels` will be invalid. (Maybe allocate from the Pinned object heap and release later)

Reply:
Nope, stackalloc'd memory isn't freed until the method exits.